app/views/layouts/application.html.haml

```diff
@@ -1,10 +1,9 @@
-!!!
-%html
-  %head
-    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
-    %title MoneyToPrisoners
-    = stylesheet_link_tag    'application', media: 'all'
-    = javascript_include_tag 'application'
-    = csrf_meta_tags
-  %body
-    = yield
+- content_for :page_title, 'Digital payments'
+
+- content_for :global_subheader do
+  .user-details
+    %span Logged in as Emma Wilkes
+    %span
+      %a(href="#") Log out
```

Comment:
:+1: 

The spans separate the elements in order to create the desired pipe in the design. Will think about making it more semantic.